Sources/MembraneRTC/Events/Event.swift

```diff
@@ -30,12 +30,12 @@ public protocol ReceivableEvent {
     var type: ReceivableEventType { get }
 }
 
-internal struct ReceivableEventBase: Decodable {
+public struct ReceivableEventBase: Decodable {
```

Comment:
why public? it was intentionally `internal` in order to not expose this to the user and to generate docs nicely
this also applies to the other places